frontend/src/app/commonComponents/Header.tsx

```diff
@@ -108,6 +108,14 @@ const Header: React.FC<{}> = () => {
     );
   };
 
+  const activeNavItem = "active-nav-item prime-nav-link";
+  const inactiveNavItem = "prime-nav-link";
+  const getNavItemClassName = ({ isActive }: { isActive: boolean }) =>
+    isActive ? activeNavItem : inactiveNavItem;
+  const getNavItemStyle = ({ isActive }: { isActive: boolean }) => ({
```

Comment:
adding this comment to put these styles into the css class as discussed in our walkthrough

Reply:
Done!